includes/admin/class-wp-job-manager-settings.php

```diff
@@ -322,7 +322,7 @@ public function output() {
 								break;
 								case "password" :
 
-									?><input id="setting-<?php echo $option['name']; ?>" class="regular-text" type="password" name="<?php echo $option['name']; ?>" value="<?php esc_attr_e( $value ); ?>" <?php echo implode( ' ', $attributes ); ?> <?php echo $placeholder; ?> /><?php
+									?><input id="setting-<?php echo $option['name']; ?>" class="regular-text" type="password" name="<?php echo $option['name']; ?>" value="<?php echo esc_attr( $value ); ?>" <?php echo implode( ' ', $attributes ); ?> <?php echo $placeholder; ?> /><?php
```

Comment:
As these are form inputs, I don't see any point in translating a form value that the user then can edit and save -- wouldn't they just change the form field directly and then save it?  If the user wanted to use WPML to translate the option for different front-end views, that's fine I guess, but that would have nothing to do with how it's displayed in a field on the back-end, as it'd be like ... the original value gets translated on the first save, then the translation gets passed in to get translated on the second?  I don't think it's a workable use case in this instance.